README.md

```diff
@@ -222,10 +228,10 @@ puppet preview --last --view overview --nodes diff_nodes
 ####Process output
 
 All output (except reports intended for human use) is written in JSON format to allow further processing with tools like 'jq' (JSON query). The output is written to a subdirectory named after the node of the directory appointed
-by the setting `preview_outputdir` (defaults to `$vardir/preview`):
+by the option `--preview-outputdir` (defaults to `$vardir/preview`):
```

Comment:
IIRC, this is a setting - which is important to know since you can configure it instead of always giving it as an option.